.pylintrc

```diff
@@ -32,7 +32,7 @@ good-names=e,_,d,f,i,l,p,w,fn,fs,id,pc,sc,zf,setUp,tearDown,longMessage,maxDiff
 dummy-variables-rgx=_|unused_*
 
 # Regular expression which should match good variable names
-variable-rgx=[a-z_][a-z0-9_]*$
+variable-rgx=^(?:([a-z][a-z0-9]*)|(_[a-z][a-z0-9]*))(_[a-z0-9]+)*$
```

Comment:
This looks a bit complicated. FYI, pylint allows using `<snake>`  in the regex, can we make it simpler using that?

Reply:
Done.

Reply:
Also the builtin-snake case naming convention allows us to have variable names with trailing underscores, like `property_` which I have changed in 2 files - should I revert back those names or are those changes fine?

Reply:
Please forbid trailing underscores -- we don't use these, as a rule. Thanks!

Reply:
I have used built-in snake case convention, and it does allow to have trailing underscore. You mean I should make changes to forbid that?

Reply:
Hi @sajalasati, Thanks for checking this, I'm fine with having the regex you have written but just too simplify it can check whether we can use `<snake>` in the regex to make it simpler? (You can check [this](http://pylint.pycqa.org/en/latest/user_guide/options.html?highlight=snake#cmdoption-inlinevar-rgx) to see how to use <snake> in the regex)

Reply:
I read [this comment](https://github.com/oppia/oppia/pull/9659#issuecomment-657114639) and it gives answer to my question! Thanks for all the work, @sajalasati!

Reply:
Ok, that's good! Thanks!